starter-code/index.html

```diff
@@ -0,0 +1,257 @@
+                <p>Etiam mauris tortor, pharetra quis lobortis in, pharetra in diam</p>
+                <p>POSTED ON AUGUST 19, 2013 BY ADMIN</p>
+                <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+              </div>
+              <div class="col-md-3">
+                <img class="latestProjectsImage" src="img/project-2.jpg" alt="">
+                <p>Falauris sollicitudin quis dolor venenatis facilisis</p>
+                <p>POSTED ON AUGUST 19, 2013 BY ADMIN</p>
+                <p>Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam! Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non ut magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum bibendum […]</p>
+              </div>
+              <div class="col-md-3">
+                <img class="latestProjectsImage" src="img/project-3.jpg" alt="">
+                <p>Suspendisse ornare, felis feugiat suscipit pharetra</p>
+                <p>POSTED ON AUGUST 19, 2013 BY ADMIN</p>
+                <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra, lorem est viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+              </div>
+              <div class="col-md-3">
+                      <img class="latestProjectsImage" src="img/project-4.jpg" alt="">
+                <p>Festibulum purus odio, ornare non posuere</p>
+                <p>POSTED ON AUGUST 19, 2013 BY ADMIN</p>
+                <p>Aenean quis rhoncus nisl, eget sollicitudin ipsum. Curabitur a elementum purus. Quisque vitae dui at arcu porttitor laoreet ac vitae nibh. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nulla facilisi. Nulla egestas at nunc at volutpat. Nullam sodales at diam vel tempor. Sed pretium, leo vel pellentesque dictum, ligula […]</p>
+              </div>
+              
+            </div>
+            <!-- 4 columns -->
+            
+            <!-- footer -->
+            <footer>
+              <div class="row">
+                
+                <div class="col-md-3">
+                  <img class="circle-image" src="img/01-circle.jpg" alt="">
```

Comment:
your src for the image is not found. You don't have a 01-circle.jpg in y9our img folder.